src/rno-g.h

```diff
@@ -183,6 +193,8 @@ typedef struct rno_g_waveform
   int16_t radiant_waveforms[RNO_G_NUM_RADIANT_CHANNELS][RNO_G_MAX_RADIANT_NSAMPLES]; //unrolled. 
   uint8_t lt_waveforms[RNO_G_NUM_LT_CHANNELS][RNO_G_MAX_LT_NSAMPLES]; // 8-bit digitizer 
   uint8_t station; 
+  radiant_readout_delay_t radiant_readout_delays;
```

Comment:
for similar reasons as before, I would put the uint16_t first since I think it will waste less space padding